src/Config/web.config.php

```diff
@@ -23,4 +23,38 @@
 return [
     'version'                  => '3.0.20',
     'queue_status_update_time' => 10 * 1000, // milliseconds = seconds x 1,000
+    'esi_roles_map'             => [
```

Comment:
Any reason why this was moved? I prefer not to put static stuff like this in configuration files. We could maybe have a separate map file for this if needed.

Reply:
It's mostly due to class refactoring and I got the filling it was almost a kind of configuration stuff.
If we're putting this map in another file, in which namespace should it be put ?

Reply:
I suggest `Seat\Web\Acl`.

Reply:
addressed with https://github.com/warlof/eveseat-web/blob/acl-ui-revamp/src/Acl/EsiRoleElement.php and https://github.com/warlof/eveseat-web/blob/acl-ui-revamp/src/Acl/EsiRolesMap.php